frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -249,3 +251,48 @@ export function FunnelInvalidExclusionFiltersEmptyState(): JSX.Element {
         </div>
     )
 }
+
+export function SavedInsightsEmptyState(): JSX.Element {
+    const { addGraph } = useActions(savedInsightsLogic)
+    const { tab } = useValues(savedInsightsLogic)
+
+    return (
+        <div className="saved-insight-empty-state">
+            <div className="insight-empty-state__wrapper">
+                <div className="illustration-main">
+                    <TrendUp />
+                </div>
+                <h2 className="empty-state__title">
+                    {
+                        {
+                            [`${SavedInsightsTabs.All}`]: 'There are no insights for this project',
+                            [`${SavedInsightsTabs.Yours}`]: "You haven't created insights for this project",
+                            [`${SavedInsightsTabs.Favorites}`]: 'There are no favorited insights for this project',
+                        }[tab]
+                    }
+                </h2>
+                <p className="empty-state__description">
+                    {
+                        {
+                            [`${SavedInsightsTabs.All}`]: 'Once you create an insight, it will show up here.',
+                            [`${SavedInsightsTabs.Yours}`]: 'Once you create an insight, it will show up here.',
+                            [`${SavedInsightsTabs.Favorites}`]: 'Once you favorite an insight, it will show up here.',
```

Comment:
Just one nit: could these be stored as `Record<SavedInsightsTabs, string>` constants _outside_ of the component?

Reply:
sure 👍